tests/physics/helicity_formalism/angular_distributions/epem_to_dm_d0_pip_decay/model.yml

```diff
@@ -0,0 +1,172 @@
+Kinematics:
```

Comment:
Isn't it better to generate this model? That would
1. allow for testing more channels with the same test procedure (and therefore also avoid having all these `__init__.py` files)
2. avoid having to rewrite the dumped model once the `expertsystem` changes its dump format
3. avoid writing to disk (which causes issues sometimes when running `pytest -n auto`)